openstack/er/v3/associations/requests.go

```diff
@@ -0,0 +1,98 @@
+package associations
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// ModifyOpts is the structure used to modify the association under a specified route table.
+type ModifyOpts struct {
```

Comment:
```suggestion
type AssociateOpts struct {
```

Reply:
CreateOpts and DeleteOpts are used to keep names consistent.